clusters/cijioagents2.yaml

```diff
@@ -0,0 +1,59 @@
+---
+helmDefaults:
+  atomic: true
+  force: false
+  timeout: 300
+  wait: true
+repositories:
+  # https://github.com/DataDog/helm-charts/
+  - name: datadog
+    url: https://helm.datadoghq.com
+  # https://github.com/jenkins-infra/helm-charts/
+  - name: jenkins-infra
+    url: https://jenkins-infra.github.io/helm-charts
+releases:
+  - name: docker-registry-secrets
+    # This helm chart doesn't create any resources within the namespace specified below.
+    # Specifying a namespace is required by the "needs" feature of helmfile (to allow referencing to this release from others)
+    namespace: default
+    chart: jenkins-infra/docker-registry-secrets
+    version: 0.1.0
+    values:
+      - "../config/docker-registry-secrets.yaml"
+    secrets:
+      - "../secrets/config/docker-registry-secrets/secrets.yaml"
+  - name: datadog
+    needs:
+      - default/docker-registry-secrets
+    namespace: datadog
+    chart: datadog/datadog
+    version: 3.83.0
```

Comment:
Don't forget to update the `updatecli` manifest (prepare a draft PR or add yourself an issue)